tests/L0/run_instance_norm_nvfuser/test_instance_norm_nvfuser.py

```diff
@@ -76,3 +79,24 @@ def test_sweep(self):
                 self.affine = affine
                 self.init_modules()
                 self.check_same_output() 
+
+    @unittest.skipIf(torch.cuda.device_count() < 2, "more than 1 GPU required")
+    def test_multigpu(self):
+        class Model(nn.Module):
+            def __init__(self):
+                super(Model, self).__init__()
```

Comment:
super nit-picking to super() to use python3 syntax
```suggestion
                super().__init__()
```